### PR TITLE
Refactor Linked Data Parser

### PR DESCRIPTION
**Pull request by Cj-Malone**
Also added a few spiders, Chef & Brewer, Farmhouse Inns and Go Outdoors.

### CODE REVIEW

**Comment by Cj-Malone** [comment]
We should consider setting a default [download_delay](https://docs.scrapy.org/en/latest/topics/settings.html?highlight=delay#download-delay). Maybe 1 second would be polite? Individual spiders could still increase or decrease it.

**Comment by iandees** [comment]
Yea, a default sounds good. There's a max run time for each spider in a global run that might need to be tweaked, but something between 0.75 and 1.0 seems like a pretty polite thing to default to.

File: locations/spiders/farmhouse_inns.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        "brand_wikidata": "Q105504972",
+        "country": "GB",
+    }
+    sitemap_urls = ["https://www.farmhouseinns.co.uk/xml-sitemap"]
+    sitemap_rules = [
+        (r"https:\/\/www\.farmhouseinns\.co\.uk\/pubs\/([-\w]+)\/([-\w]+)\/$", "parse")
+    ]
+
+    def parse(self, response):
+        yield LinkedDataParser.parse(response, "BarOrPub")
```
Comment (iandees) [inline locations/spiders/farmhouse_inns.py:19]: Nice :+1:

**Comment by iandees** [comment]
Thanks, this looks great.

**Pull request merged**